treeview.lua attempt to index a nil value error

I sometimes can't create file within lite-xl because of the error below

Steps to reproduce on master branch:

1. open lite-xl in an empty folder
2. create a subdirectory A, and create a subdirectory B in the folder A.
3. attempt to create file in folder B

```
Wed Nov 19 18:51:34 2025 [ERROR] /home/av/lite-xl/data/plugins/treeview.lua:580: attempt to index a nil value (local 'item') at /home/av/lite-xl/data/core/init.lua:847

stack traceback:
/home/av/lite-xl/data/plugins/treeview.lua:580: in upvalue 'is_project_folder'
/home/av/lite-xl/data/plugins/treeview.lua:616: in field 'predicate'
/home/av/lite-xl/data/core/command.lua:120: in function 'core.command.get_all_valid'
/home/av/lite-xl/data/core/commands/core.lua:197: in field 'perform'
/home/av/lite-xl/data/core/command.lua:147: in function </home/av/lite-xl/data/core/command.lua:137>
[C]: in function 'xpcall'
/home/av/lite-xl/data/core/init.lua:846: in function 'core.try'
/home/av/lite-xl/data/core/command.lua:169: in function 'core.command.perform'
/home/av/lite-xl/data/core/keymap.lua:223: in function 'core.keymap.on_key_pressed'
/home/av/lite-xl/data/core/init.lua:867: in upvalue 'on_event'
/home/av/lite-xl/data/plugins/macro.lua:19: in function 'core.on_event'
[C]: in function 'xpcall'
/home/av/lite-xl/data/core/init.lua:846: in function 'core.try'
/home/av/lite-xl/data/core/init.lua:941: in function 'core.step'
/home/av/lite-xl/data/core/init.lua:1034: in function 'core.run'
(...tail calls...)
[string "local core..."]:14: in function <[string "local core..."]:6>
[C]: in function 'xpcall'
[string "local core..."]:6: in main chunk
```

--- a/data/plugins/treeview.lua
+++ b/data/plugins/treeview.lua
@@ -835,7 +835,7 @@ command.add(
     local text
     if not is_project_folder(item) then
       if item.type == "dir" then
-        text = item.project:normalize_path(item.filename) .. PATHSEP
+        text = item.project:normalize_path(item.abs_filename) .. PATHSEP
       elseif item.type == "file" then
         text = item.project:normalize_path(common.dirname(item.abs_filename)) .. PATHSEP
       end

